v2/emailpassword/common-customizations/email-verification/about.mdx

```diff
@@ -156,12 +156,12 @@ SuperTokens.init({
 
 <Answer title="Prebuilt UI">
 
-- In `supertokensAuthComponent.tsx` (where we initialize the `supertokens-auth-react` SDK) add the `EmailVerification` recipe to the `recipeList`.
+```tsx
+import { Component } from "@angular/core";
 
-```tsx title="/app/auth/supertokensAuthComponent.tsx "
 import SuperTokens from "supertokens-auth-react";
-import EmailVerification from "supertokens-auth-react/recipe/emailverification";
 import Session from "supertokens-auth-react/recipe/session";
+import EmailVerification from "supertokens-auth-react/recipe/emailverification";
 
```

Comment:
Instead of repeating the code snippet here, we can just use `~COPY-TABS=reactjs`? (same goes for vue js)